bin/ch/Helpers.cpp

```diff
@@ -370,7 +370,7 @@ HRESULT Helpers::LoadBinaryFile(LPCSTR filename, LPCSTR& contents, UINT& lengthB
             }
 #endif
             fprintf(stderr, "\n");
-            IfFailGo(E_FAIL);
+            return E_FAIL;
```

Comment:
Looks like the `return E_FAIL` can be moved out a level and eliminate the `else` condition?